src/components/GameGrid/GameBoard.tsx

```diff
@@ -32,11 +37,46 @@ const GameBoard: React.FC<Props> = ({ gameData }: Props) => {
       <TextContainer>
         <Shout>Other games</Shout>
 
-        <Loud>Knockout rounds</Loud>
+        <Loud>Round of 32</Loud>
```

Comment:
Personally, I think the rounds should be the other way round: Finals should be at the top (with 3rd place, then semi finals after, etc), with Round of 32 at the bottom.  I prefer this because I think it flows better information wise, since the latest information will be at the top. It's the same structure that was used for the Tube World Cup site (https://gum-joe.github.io/tube-world-cup/).

However, @davwheat may have a different opinion, so I'm unsure if this should be actioned upon.

Reply:
I put them this way round to begin with while the Round of 32 is currently in progress with the idea of as the competition progresses the 'active' round moving up to the top.

Reply:
Hmm Yeah that probably makes more sense (since otherwise the first thing a viewer sees is a bunch of question marks).

I think it makes most sense to move up to the top as we go along like you said, such that by the end of the compeition the top of the page is the final, then the 3rd place playoff, then the semi-final, etc.  In which case this is find as-is for now, will suggest adding a comment though to note that.

Reply:
(have added a comment as a suggestion)